src/views/requestAccess.js

```diff
@@ -0,0 +1,165 @@
+import React, { useState } from "react";
+import { useHistory } from 'react-router-dom';
+import { ACCESS_REQUEST_DATA } from '../components/DashboardModule/data';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSearch } from '@fortawesome/free-solid-svg-icons';
+
+
+export const Footer = () => {
+	return (
+		<footer className="dashboard__footer">
+            <p className="dashboard__footer_logo_text">
+                <span className="bold">JOIN</span> 2018
+            </p>
+        </footer>
+	);
+}
+
+export const DropDownItem = ({dropContent, clickHandle}) => {
+	return (
+		<div className="dropdown-item">
+	      	<label className="radio">
+		      	<input type="radio" name="answer" onClick={clickHandle} className="radio-dropdown"></input>
+		      	{dropContent}
+		    </label>
+	    </div>
+	);
+}
+
+export const PropertyPage = () => {
```

Comment:
Can we rename `PropertyPage` and `JoinPage` to something like `PropertyManagerOptions` and `JoinStaffOptions`? They aren't pages so it's a little misleading